spec/forms/assessor_interface/assessment_declaration_decline_form_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+RSpec.describe AssessorInterface::AssessmentDeclarationDeclineForm,
+               type: :model do
+  let(:assessment) { create(:assessment, :started) }
+  let(:session) { {} }
+  subject(:form) { described_class.new(session:, assessment:) }
+  it { is_expected.to validate_presence_of(:declaration) }
+
+  let(:assessment) { create(:assessment, :decline_with_no_failure_Reasons) }
+  it { is_expected.to validate_presence_of(:recommendation_assessor_note) }
+
+  describe "#save" do
+    context "when valid" do
+      it "saves the form" do
+        form.declaration = true
+        form.recommendation_assessor_note = "This is a note"
```

Comment:
Worth moving these outside the expectation (`it`) as they're the setup for the test, not part of the test itself.